src/main/java/org/example/application/CountMonthlyCostsService.java

```diff
@@ -0,0 +1,30 @@
+package org.example.application;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.example.domain.customer.Customer;
+import org.example.domain.money.Currency;
+import org.example.domain.money.Money;
+import org.example.port.in.invoice.CountMonthlyCostsUseCase;
+import org.example.port.out.InvoiceRepository;
+
+import java.math.BigDecimal;
+import java.time.YearMonth;
+
+@Slf4j
+@AllArgsConstructor
+public class CountMonthlyCostsService implements CountMonthlyCostsUseCase {
+    private InvoiceRepository invoiceRepository;
```

Comment:
interfejs nie implementacja

Reply:
Here I provided the interface not the implementation.